ceph-radosgw/tests/bundles/jammy-caracal-multisite.yaml

```diff
@@ -1,101 +0,0 @@
-options:
-  source: &source cloud:jammy-caracal
-
-series: jammy
-
-comment:
-- 'machines section to decide order of deployment. database sooner = faster'
-machines:
-  '0':
-  '1':
-  '2':
-  '3':
-  '4':
-  '5':
-  '6':
-  '7':
-  '8':
-  '9':
-
-applications:
-  ceph-radosgw:
```

Comment:
although the multisite tests are removed (we are not running them anyways) I would eventually want to have the multisite tests running in github CI as well.